incubator/mssql-linux/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "mssql.fullname" . }}-secret
+  labels:
+    app: {{ template "mssql.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  password: {{ randAlphaNum 15 | b64enc | quote }}
```

Comment:
Please provide an option to explicitly set the password and only auto-generate it if not set.

Reply:
Thanks for the review.  Added